app/components/Article/Article.module.scss

```diff
@@ -0,0 +1,83 @@
+@use '@/styles/mixins.scss' as mix;
+@use '@/styles/variables.scss' as var;
+
+.article {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var.$color-700-red;
+  border-radius: var.$border-radius;
+  height: 50vh;
+
+  img {
+    height: 14rem;
+    width: 100%;
+    object-fit: cover;
+    border-top-right-radius: var.$border-radius;
+    border-top-left-radius: var.$border-radius;
+  }
+
+  .info {
+    @include mix.flex(flex-start, flex-start, column);
+    gap: var.$spacing-s;
+    padding: var.$spacing-s;
+    height: -webkit-fill-available;
+  }
+
+  .articleDescr {
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 3;
+  }
+
+  .articleAuthor {
+    align-self: flex-end;
+    font-style: italic;
+    color: var.$color-700-neutral;
+  }
+
+  .readMore {
+    display: block;
+    text-align: center;
+    background-color: var.$color-700-red;
+    color: var.$color-white;
+    font-size: larger;
+    font-weight: 700;
+    padding: var.$spacing-xs;
+    transition: var.$transition;
+    border-bottom-right-radius: var.$border-radius;
+    border-bottom-left-radius: var.$border-radius;
+
+    &:hover {
+      background-color: red;
+    }
+  }
+}
+
+@media (width <= 1080px) {
```

Comment:
1. Here is a perfect place for mixins. Please create mixins for mobile, tablet and desktop sizes. 
2. Please use media queries inside each classes, I think it’s more readable this way
3. Could you please send a link to read about such media query syntax, because I know min width and max width, so I’m curious about < > <= >=

Reply:
Links:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries
https://css-tricks.com/the-new-css-media-query-range-syntax/
https://caniuse.com/css-media-range-syntax